### PR TITLE
fix the skulpt.py bug

### PR DESCRIPTION
**Pull request by acekingke**
Signed-off-by: Ke Yuchang <keyuchang@kaka888.com>

### CODE REVIEW

File: skulpt.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def is64bit():
     return sys.maxsize > 2**32
 
 if sys.platform == "win32":
-    winbase = ".\\support\\d8\\x32"
+    winbase = ".\\support\\d8\\x32_"
```
Comment (albertjan) [inline skulpt.py]: I see why you did this but, I think you should change this to `".\\support\\d8.exe"` and remove line 179 and 180. I don't see the need of this 32 bit check.

File: src/pgen/parser/tokenize.py [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
 from codecs import BOM_UTF8, lookup
-from lib2to3.pgen2.token import *
+#from lib2to3.pgen2.token import *
 
 import token
 __all__ = [x for x in dir(token) if x[0] != '_'] + ["tokenize",
            "generate_tokens", "untokenize"]
 del token
-
+from token  import *
```
Comment (albertjan) [inline src/pgen/parser/tokenize.py]: Remove the extra space though

**Comment by albertjan** [comment]
Hi @acekingke can you revert the change in the package.json, and just call `.\skulpt.py` on windows  directly, we need to figure out a better way to do this on both windows and mac. 

@j-f1 do you know how we could solve this in a cross platform way? Maybe install skulpt in `./node_modules/.bin` (but I don't know how)

**Comment by j-f1** [comment]
You can do `python skulpt.py`, which will look for `skulpt.py` in the current directory.

**Comment by albertjan** [comment]
@j-f1 yeah 😄 ... but then we have the whole python version thing that screws us up.

**Comment by bnmnetp** [comment]
@albertjan Is this still relevant or should we close it?

**Comment by bnmnetp** [comment]
Thanks, this is superseded by the ongoing work with node.

**Pull request closed without merging**